feature/watch.go

```diff
@@ -2,11 +2,27 @@ package feature
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"io"
 	"log"
+	"os"
 	"path/filepath"
+	"sync"
+	"time"
 
 	"github.com/fsnotify/fsnotify"
+
+	"github.com/ajm188/go-ff/internal/jsonutil"
+)
+
+var (
+	watchM      sync.Mutex
+	watching    = false
+	watchedFile = ""
```

Comment:
Change these to rely on zero values

```suggestion
	watching    bool
	watchedFile string
```